client/lib/purchases/types.ts

```diff
@@ -270,9 +277,103 @@ export interface MembershipSubscriptionsSite {
 	subscriptions: MembershipSubscription[];
 }
 
-export type GetChangePaymentMethodUrlFor = ( siteSlug: string, purchase: Purchase ) => string;
-
 export interface Owner {
 	ID: string;
 	display_name: string;
 }
+export type GetChangePaymentMethodUrlFor = ( siteSlug: string, purchase: Purchase ) => string;
+export type GetManagePurchaseUrlFor = ( siteSlug: string, attachedToPurchaseId: string ) => string;
+
+export type RenderRenewsOrExpiresOn = ( args: {
+	moment: ReturnType< typeof useLocalizedMoment >;
+	purchase: Purchase;
+	siteSlug: string | undefined;
+	translate: ReturnType< typeof useTranslate >;
+	getManagePurchaseUrlFor: GetManagePurchaseUrlFor;
+} ) => string;
+
+export type RenderRenewsOrExpiresOnLabel = ( args: {
+	purchase: Purchase;
+	translate: ReturnType< typeof useTranslate >;
+} ) => string;
+
+export interface Site {
+	ID?: number;
+	name?: string;
+	description?: string;
+	URL?: string;
+	capabilities?: { [ key: string ]: boolean };
+	jetpack?: boolean;
+	is_multisite?: boolean;
+	site_owner?: number;
+	lang?: string;
+	visible?: boolean;
+	is_private?: boolean;
+	is_coming_soon?: boolean;
+	single_user_site?: boolean;
+	is_vip?: boolean;
+	options?: Options;
+	plan?: Plan;
+
+	jetpack_modules?: null;
+	launch_status?: string;
+	site_migration?: null;
+	is_core_site_editor_enabled?: boolean;
+	is_wpcom_atomic?: boolean;
+	user_interactions?: Date[];
+	domain?: string;
+	slug?: string;
+	title?: string;
+}
+
+export interface Options {
```

Comment:
I wonder: is this the same type as `SiteDetailsOptions` in `packages/data-stores/src/site/types.ts`? If so, we can probably re-use that and import it here.